python/deepgram-transcribe-video/main.py

```diff
@@ -0,0 +1,30 @@
+from deepgram import Deepgram
+import asyncio
+import json
+
+
+async def transribeVideo(api_key, payload):
+    dg_client = Deepgram(api_key)
+
+    source = {'url': payload['fileUrl']}
+
+    try:
+        response = await dg_client.transcription.prerecorded(source, {'punctuate': True, "model": 'video'})
+    except Exception as e:
+        return {'success': False, 'message': str(e)}
+
+    return {'success': True, 'deepgramData': response}
+
+
+def main(req, res):
+
+    payload = json.loads(req.payload)
+
+    api_key = req.variables['DEEPGRAM_API_KEY']
+
+    if not api_key:
+        res.json({'success': False, 'message': 'DEEPGRAM_API_KEY not set'})
+
+    rs = asyncio.run(transribeVideo(api_key, payload))
+
+    return res.json(rs)
```

Comment:
Please add error handling to catch all errors so that your code doesn't throw any exceptions. You should be calling res.json() and returning a proper error response as defined in the issue.

For example, not passing an environment variable results in an unhelpful error:

```shell
curl http://localhost:3000/ -d '{"variables": {},"payload": "{\"fileUrl\":\"https://rawcdn.githack.com/deepgram-devs/transcribe-videos/62fc7769d6e2bf38e420ee5224060922af4546f7/deepgram.mp4\" }"
> }' -H "X-Internal-Challenge: secret-key" -H "Content-Type: application/json"
<!doctype html>
<html lang=en>
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```